Documentation/kubernetes/install.rst

```diff
@@ -4,6 +4,11 @@
 Installation Guide
 ******************
 
+.. note:: This is the detailed installation guide aimed at production
+          installations. If you are looking to get started quickly, the
+          :ref:`gs_minikube:` or the :ref:`k8s_quick` guide may be better
+          options.
```

Comment:
The `gs_minikube` reference will not render properly because of the extra character `:`.

```diff
diff --git a/Documentation/kubernetes/install.rst b/Documentation/kubernetes/install.rst
index cf38b17ce151..fb35220d5b42 100644
--- a/Documentation/kubernetes/install.rst
+++ b/Documentation/kubernetes/install.rst
@@ -6,7 +6,7 @@ Installation Guide
 
 .. note:: This is the detailed installation guide aimed at production
           installations. If you are looking to get started quickly, the
-          :ref:`gs_minikube:` or the :ref:`k8s_quick` guide may be better
+          :ref:`gs_minikube` or the :ref:`k8s_quick` guide may be better
           options.
 
 This section describes how to install and run Cilium on Kubernetes. The
```

Reply:
Fixed